independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InjectableInstance.java

```diff
@@ -24,4 +25,12 @@
     @Override
     <U extends T> InjectableInstance<U> select(TypeLiteral<U> subtype, Annotation... qualifiers);
 
+    /**
+     * Removes the cached result of the {@link #get()} operation. If the cached result was a contextual reference of
```

Comment:
```suggestion
     * Removes all cached results of the {@link #get()} operations. If any of cached results was a contextual reference of
```

Reply:
There is only one cached result of `get()`...

Reply:
Yeah, my bad, sorry.